### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by fsoubelet**
Small things:
- fixing of typos,
- more consistent formatting,
- removal of white space in folder and file names,
- a bit more details in some READMEs,
- fixing the main README's license section,
- adding a .gitignore file.

Main changes are in the AC dipole tracking script where a more commented / explained version replaced the previous one.

### CODE REVIEW

File: LHC/Injection_Energy_with_Misalignments_and_Correction/README.md [hunk 1]
```diff
@@ -3,3 +3,5 @@ Misalignments are applied and orbit is corrected.
 Nonlinear errors are applied and corrections applied as were used in RunII.
 A1/A2/B1/B2 are excluded as these are assumed to have been well corrected and residual betabeating and
 coupling stems for misalignments and feed down.
+
+Set `NRJ`?
```
Comment (mihofer) [inline LHC/Injection_Energy_with_Misalignments_and_Correction/README.md:7]: NRJ should be kept a t 450 as technically this script is only valid for injection energy, at top energy the magnetic errors of the triplet play a bigger role and nonlinear correctors should be set too, which is not done in this script (but there is an open issue to add a script for that in MESS)

File: LHC/AC_Dipole_Tracking/job.madx [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+SELECT_PTC_NORMAL, Q1=0, Q2=0;
+SELECT_PTC_NORMAL, DQ1=1, DQ2=1; ! First order chromaticities (dQ1/ddeltap^1)
+SELECT_PTC_NORMAL, DQ1=2, DQ2=2; ! Second order chromaticities (dQ1/ddeltap^2)
+SELECT_PTC_NORMAL, DQ1=3, DQ2=3; ! Third order chromaticities (dQ1/ddeltap^3)
+SELECT_PTC_NORMAL, ANHX=1,0,0; ! dQx / d2Jx
+SELECT_PTC_NORMAL, ANHY=0,1,0; ! dQy / d2Jy
+SELECT_PTC_NORMAL, ANHX=0,1,0; ! dQx / d2Jy
+SELECT_PTC_NORMAL, ANHY=1,0,0; ! dQy / d2Jx
+SELECT_PTC_NORMAL, ANHX=2,0,0; ! d^2Qx / d2Jx^2
+SELECT_PTC_NORMAL, ANHX=1,1,0; ! d^2Qx / d2JxdJy
```
Comment (mihofer) [inline LHC/AC_Dipole_Tracking/job.madx:193]: should be d2Jx d2Jy

**Review by mihofer — APPROVED** [review]
good to go from my side, comments should be corrected

File: LHC/Injection_Energy_with_Misalignments_and_Correction/README.md (hunk 1 above)
Comment (fsoubelet) [reply LHC/Injection_Energy_with_Misalignments_and_Correction/README.md:7]: Good catch.

File: LHC/AC_Dipole_Tracking/job.madx (hunk 2 above)
Comment (fsoubelet) [reply LHC/AC_Dipole_Tracking/job.madx:193]: Another good catch.

File: LHC/Injection_Energy_with_Misalignments_and_Correction/README.md [hunk 3]
```diff
@@ -2,4 +2,4 @@ Script for creating a thin LHC beam 1 model with misalignments and nonlinear err
 Misalignments are applied and orbit is corrected.
 Nonlinear errors are applied and corrections applied as were used in RunII.
 A1/A2/B1/B2 are excluded as these are assumed to have been well corrected and residual betabeating and
-coupling stems for misalignments and feed down.
+coupling stems for misalignments and feed down.
```
Comment (mihofer) [inline LHC/Injection_Energy_with_Misalignments_and_Correction/README.md:5]: stems from

**Pull request merged**